src/components/Home.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchBrands } from '../Redux/Slices/brandsFetchSlice';
+import { setTitle } from '../Redux/Slices/menuSlice';
+import Brands from './Brands';
+
+function Home() {
+  const dispatch = useDispatch();
+  const { brands } = useSelector((state) => state.brands);
+  const { loading } = useSelector((state) => state.brands);
+  useEffect(() => {
+    dispatch(fetchBrands());
+    dispatch(setTitle('Brands'));
+  }, []);
+
+  const handleBrandClick = (link) => {
+    dispatch(fetchBrands(link));
+  };
+
+  if (loading) {
+    return (
+      <h2 className="h-screen animate-pulse text-white flex items-center justify-center text-xl">
+        Loading.........
+      </h2>
+    );
+  }
+  return (
+    <main className="">
+      <article className=" homeBg flex  flex-col text-white items-end p-8">
+        <h3 className="flex flex-col text-2xl font-bold items-end">
+          2023
+          <span>Phone Specs</span>
+        </h3>
+        <h4 className="flex flex-col items-center">
+          {`${brands.length}
+          Brands`}
+        </h4>
+      </article>
+      <h3 className="text-white px-3 py-1 border bg-rose-500 border-[#e0407d] text-xl shadow-xl">
+        All Brands Stats
```

Comment:
- [ ] You did really a great job on this project but you missed a major requirement. Please add a feature on the home page to filter Brands. (You should include a search feature on the page).
![image](https://user-images.githubusercontent.com/83803414/193900059-ae0b963d-7671-4e1c-910c-bc3ad366aaa2.png)